boot/bootdata/hivedef.inf

```diff
@@ -289,7 +289,7 @@ HKCU,"Control Panel\Appearance\New Schemes\0\Sizes\0","Size #6",0x000b0001,0c,00
 HKCU,"Control Panel\Appearance\New Schemes\0\Sizes\0","Size #7",0x000b0001,12,00,00,00,00,00,00,00
 HKCU,"Control Panel\Appearance\New Schemes\0\Sizes\0","Size #8",0x000b0001,12,00,00,00,00,00,00,00
 ; Brick
-HKCU,"Control Panel\Appearance\New Schemes\1","DisplayName",0x00020000,"@themeui.dll,-851"
+HKCU,"Control Panel\Appearance\New Schemes\1","DisplayName",0x00020000,"@themeui.dll,-850"
```

Comment:
Were the old values (851 in this case) compatible with Windows Server 2003 ones?

Reply:
No, in Windows XP the value was
> @themeui.dll,-2019
But at the time, I didn't think it mattered if the value was exact or not. Want me to change it? Just keep in mind that this month is extremely busy for me.

Reply:
No problem, don't change anything. I was just wondering whether the original values were from somewhere, or whether they were just chosen arbitrarily.